src/parser/pegen.ts

```diff
@@ -1502,6 +1521,23 @@ export function seq_flatten(p: Parser, seqs: AST[][]): AST[] {
 //     return _Py_Name(uni, Load, EXTRA_EXPR(first_name, second_name));
 // }
 
+class UnreachableException extends Error {}
```

Comment:
Errors is another layer we need to think about at some point.

Reply:
Yeah I needed a error